doc/getstarted/build_and_install/build_from_source.md

```diff
@@ -15,13 +15,15 @@ cd paddle
 
 ## <span id="requirements">Requirements</span>
 
-To compile the source code, your computer must be equipped with GCC >=4.6 or Clang compiler.
+To compile the source code, your computer must be equipped with the following dependencies.
+
 ### Dependencies
```

Comment:
Given above section named "Requirements", it seems that we don't need this sub-section title "Dependencies"?